### PR TITLE
Implement settings profile pages

### PR DESCRIPTION
**Pull request by cychiuae**
ref DEV-1832
ref DEV-1833
ref DEV-1834
ref DEV-1835
ref DEV-1836
ref DEV-1837
ref DEV-1838
ref DEV-1839

### CODE REVIEW

File: authui/src/authflowv2/components/settings-user-profile-pic.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+@layer components {
+  .settings-user-profile-pic__image-container {
+    @apply relative;
+    @apply h-22 w-22 tablet:h-50 tablet:w-50;
+  }
+
+  .settings-user-profile-pic__edit-button {
+    @apply inline-flex items-center justify-center;
+    @apply w-9 h-9;
+    @apply absolute bottom-0 -right-2 tablet:right-3.5;
```
Comment (louischan-oursky) [inline authui/src/authflowv2/components/settings-user-profile-pic.css:10]: ltr trl

- This is stated in the guideline. https://github.com/authgear/authgear-server/issues/3626
- I want a stylelint rule to warn you guys automatically about this.

File: resources/authgear/templates/en/translation.json [hunk 2]
```diff
@@ -1106,8 +1112,42 @@
...
+  "v2.page.settings-profile-edit-address.default.navbar-title": "Address",
+  "v2.page.settings-profile-edit-address.default.postal-code-label": "Postal code",
+  "v2.page.settings-profile-edit-address.default.region-label": "Region",
+  "v2.page.settings-profile-edit-address.default.street-label": "Street address",
+  "v2.page.settings-profile-edit-birthdate.default.navbar-title": "Birthdate",
+  "v2.page.settings-profile-edit-gender.default.gender-label-custom": "Custom",
+  "v2.page.settings-profile-edit-gender.default.gender-label-female": "Female",
+  "v2.page.settings-profile-edit-gender.default.gender-label-male": "Male",
+  "v2.page.settings-profile-edit-gender.default.gender-label-unspecified": "Not provided",
+  "v2.page.settings-profile-edit-gender.default.title-gender": "Gender",
```
Comment (louischan-oursky) [inline resources/authgear/templates/en/translation.json:1126]: I guess this is page title? Then just `.title`, not `.title-gender`.

**Review by louischan-oursky — CHANGES_REQUESTED** [review]

File: authui/src/authflowv2/components/navbar.css [hunk 3]
```diff
@@ -69,9 +69,12 @@
     font-weight: var(--navbar__item-font-weight);
     letter-spacing: var(--navbar__item-letter-spacing);
     line-height: var(--navbar__item-line-height);
-  }
 
-  .navbar__item_icon {
-    font-size: var(--navbar__item-icon-font-size);
+    &::before {
+      /* arrow_back_ios (<) in ltr and arrow_forward_ios (>) in rtl */
+      @apply ltr:content-['arrow\_back\_ios'] rtl:content-['arrow\_forward\_ios'];
```
Comment (cychiuae) [inline authui/src/authflowv2/components/navbar.css:75]: ### LTR
Screenshot 2024-09-06 at 2 37 23 PM

### RTL
Screenshot 2024-09-06 at 2 37 41 PM

**Comment by louischan-oursky** [comment]
SCR-20240906-onjl
SCR-20240906-onkq

When the gender is custom, it is shown as Not Provided.

---

SCR-20240906-onmo

The date input handle is not shown. The end-user has no clue that the widget can be expanded by clicking at the far right.

---
SCR-20240906-onqd

The icon is 32x24. This does not work when the layout is rtl. I think the icon should be 24x24, and we apply gap 8px

File: authui/src/authflowv2/components/date-input.css [hunk 4]
```diff
@@ -1,7 +1,13 @@
 @layer components {
+  :root {
+    --date-input--calendar-icon-filter:;
```
Comment (louischan-oursky) [inline authui/src/authflowv2/components/date-input.css:3]: `: initial` or `: unset`? Empty seems weird.

**Review by louischan-oursky — CHANGES_REQUESTED** [review]

**Pull request merged**